.travis.yml

```diff
@@ -18,12 +18,12 @@ install:
   - bundle install --retry=3
 
 rvm:
-  - 1.9
-  - 2.0
+  - 1.9.3
```

Comment:
Since Ruby 1.9.2 is no longer supported, I think it’s better to be explicit about this version number. Specifying a teeny digit is also necessary for https://github.com/YorickPeterse/oga/pull/157, since Travis hasn’t defined an `rvm alias` for `2.3`.